.github/workflows/ci-workflow.yml

```diff
@@ -46,10 +46,11 @@ jobs:
         if: steps.audit.outputs.exit_code < 7
         run: yarn build
 
-      - name: Package VSIX & Upload Artifact
+      - name: Package & Upload Artifact
         if: steps.audit.outputs.exit_code < 7
+        run: yarn copy-into-build-context
         run: yarn workspace ${{ env.PROJECT_NAME }} package
       - uses: actions/upload-artifact@v2
         with:
-          name: ${{ env.PROJECT_NAME }}
+          name: ${{ env.PROJECT_NAME }}.vsix
```

Comment:
It's going to copy the artifact into a location with this name. It should have the prefix vsix, right? Because it's a vsix file.

Reply:
Yep, it is a minor fix, but for me such a name with extension is much easier to understand, when you are in the GH workflow view :)

Reply:
Please can we leave it as VSIX? So we know it has been packaged. When I download it I can see what it is exactly.

Reply:
I'm sorry, maybe I misunderstood the comment. Let me explain...

**Here is the artifact name from master branch:**
![Screenshot 2021-07-15 at 10 47 42](https://user-images.githubusercontent.com/74672982/125758981-b7b9f565-3b29-421f-8d6f-3f0b0f4d7342.png)

**Here is the artifact name from this branch (with updates):**
![Screenshot 2021-07-15 at 10 48 05](https://user-images.githubusercontent.com/74672982/125759053-424c039e-03ea-4781-9cef-4a9bb7af4308.png)

As you can see, **the '.vsix' file extension is presented in the updated version**, but not in the master. I believe, that file name with extension (.vsix) will be much more understandable. Does it make sense?

Reply:
Yes, I think it's fine. Thanks!